timesheets/overview/get_started.rst

```diff
@@ -0,0 +1,66 @@
+===========================
+Get Started with Timesheets
+===========================
+
+Filling up timesheets allows you to accurately track the time spent on tasks, making sure your
```

Comment:
I would flesh this out a bit more. 

I'd say something about timesheet allows you to bill for the amount of work spent, not more, not less, that you can add description of what you worked on. I'd split the last part with the devices to another line.